main.go

```diff
@@ -178,11 +178,12 @@ func main() {
 		Scheme: mgr.GetScheme(),
 	}
 
-	loggingSecrets := loggingcredentials.Resource{
+	loggingSecrets := credentials.Resource{
```

Comment:
That should be named credentialSecrets instead of loggingSecrets right?

Reply:
Yes, you have a point :+1: